src/ng/compile.js

```diff
@@ -1458,11 +1429,13 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
 
         nodeName = nodeName_(this.$$element);
 
-        if ((nodeName === 'a' && (key === 'href' || key === 'xlinkHref')) ||
-            (nodeName === 'img' && key === 'src')) {
-          // sanitize a[href] and img[src] values
-          this[key] = value = $$sanitizeUri(value, key === 'src');
```

Comment:
Afaict, this change means that people using `attrs.$set(key, value)` will **not** benefit from sanitization (for `<a href/xlink:href` or `<img src/ng-src`) any more. Is that correct?